src/main/java/com/thealgorithms/datastructures/trees/Maximum depth of the bt.java

```diff
@@ -0,0 +1,41 @@
+package com.thealgorithms.datastructures.trees;
+import java.util.*;
+/*
+Creating class Node for tree which contain data part,left and right of that tree
+*/
+class Node{
```

Comment:
Use the [SimpleTreeNode](https://github.com/TheAlgorithms/Java/blob/master/src/main/java/com/thealgorithms/devutils/nodes/SimpleTreeNode.java) under src.main.java.com.thealgorithms.devutils.nodes instead of creating a new class.